code/modules/antagonists/revolution/revolution.dm

```diff
@@ -397,6 +397,9 @@
 	/// List of all ex-revs. Useful because dynamic removes antag status when it ends, so this can be kept for the roundend report.
 	var/list/ex_revs = list()
 
+	/// The objective of the heads of staff, aka to kill the headrevs.
+	var/list/datum/objective/mutiny/heads_objective = list()
```

Comment:
Why not just do `for(var/datum/objective/mutiny/objective in objectives)`, rather than tracking these in a whole nother list?

Reply:
because these are different from the actual rev objectives, these are pseudo-objectives that don't actually get displayed anywhere.
They're just primarily useful for storing stateful info on the headrevs.
The objectives list contains the objective list that targets the heads of staff rather than the headrevs.

Reply:
Oh I see now